awslabs/aws-lambda-powertools-typescript
# build(deps-dev): bump esbuild from 0.14.x to 0.14.23

## Description of your changes

This PR bumps `esbuild` to its latest version. This is a dev dependency used in `examples/cdk` & in the e2e tests for each core utility and it helps `cdk` to bundle & transpile the functions code.

I've opened this PR instead of using #588 from Dependabot because the bot was not picking up all the occurrences given they had inconsistent versions.

### How to verify this change

See e2e run result: [here](https://github.com/awslabs/aws-lambda-powertools-typescript/runs/5306489061?check_suite_focus=true)

### Related issues, RFCs

[#588](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/588)

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
